src/lib/config/ModelConfig.cc

```diff
@@ -886,6 +886,18 @@ void ModelConfig::postValidation() {
   // Record any unlinked port names
   for (const auto& prt : portnames)
     invalid_ << "\t- " << prt << " has no associated reservation station\n";
+
+  // Check for valid LSQ-L1-Interface Load and Store Bandwidth
```

Comment:
A slightly more descriptive comment would be nice e.g. something along the lines of "Ensure load/store bandwidth is high enough for **vector unit**"

Reply:
Changed to make a little more clear